.github/workflows/build.yaml

```diff
@@ -71,7 +71,7 @@ jobs:
 
   build-docker:
     name: Build Docker image
-    runs-on: ubuntu-22.04
+    runs-on: ubuntu-latest
```

Comment:
we only use specific versions and will update it, once the new LTS version was released and tested.